app/javascript/onboarding/__tests__/__snapshots__/Onboarding.test.jsx.snap

```diff
@@ -86,7 +94,15 @@ preact-render-spy (1 nodes)
           class="back-button"
           type="button"
         >
-          Back
+          <svg
```

Comment:
@vaidehijoshi

Yay back button! I have some recommendations for how to architect this to improve the UX.

First of all, a 16x16 back-button icon is much too small for people to accessibly click or press without getting frustrated. I recommend:

- Increasing the width and height of the SVG icon to `24x24` pixels.
- Wrapping the SVG icon into an `<outer container>` of `40x40` pixels and make that a click/tap target.
- On the <outer container>, on hover or active, we can make the background to `var(--base-10)` to indicate the tappable area :) 

**Outer container (40x40)**
<img width="218" alt="Screenshot 2020-04-23 08 39 22" src="https://user-images.githubusercontent.com/816402/80119456-973c4f00-853e-11ea-9161-44c45fe916cd.png">

**SVG (24x24)**
<img width="160" alt="Screenshot 2020-04-23 08 39 30" src="https://user-images.githubusercontent.com/816402/80119457-97d4e580-853e-11ea-8016-cbca113e7cb2.png">

Reply:
@lisasy Thanks for bringing this up! I can work on this later today once I wrap up what I'm working on now.

> On the , on hover or active, 

I think there's a typo here, can you clarify?

Also, just for more context, I got the 16x16 size from the figma mocks; was I meant to be doing something else, or are those mocks also incorrect? Just want to know for my own edification for the next time I need to implement a design that is rendered in figma :)

Reply:
Oops, good typo catch. It is on the <outer container> that its hover or active state can show have its background changed.

```
.outer-container {
  border-radius: 100% // circle!
  background: transparent;

  &:hover,
  &:active {
    background: var(--base-10)
  }
}
```

Reply:
@vaidehijoshi In the Figma mocks, the icon elements are organized in folders, and it may be possible you exported the *vector* shape instead of its container frame (which is 24). I'll export the 24x24 SVG to you and send it to you over Slack :)